### PR TITLE
script for items table

### PR DESCRIPTION
**Pull request by thaotpnguyen**

### CODE REVIEW

File: server/database/setup.sql [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  availableTo DATE NOT NULL DEFAULT NULL,
+  address1 TEXT NOT NULL DEFAULT NULL,
+  latLng TEXT NOT NULL DEFAULT NULL,
+  photos TEXT NOT NULL DEFAULT NULL,
+  FOREIGN KEY (user_id)
+    REFERENCES users(user_id)
+    ON DELETE CASCADE
+);
+
+CREATE INDEX item_index ON items(user_id);
```
Comment (joeyohie) [inline server/database/setup.sql:34]: I'm slightly confused with the naming of this index. Is this one to index user ids? If so, maybe something like 'user index' is better? Not a big deal either way.

File: server/database/setup.sql [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+DROP TABLE IF EXISTS items;
+CREATE TABLE items (
+  item_id SERIAL PRIMARY KEY,
+  user_id INTEGER NOT NULL DEFAULT NULL,
+  title TEXT NOT NULL DEFAULT NULL,
+  category TEXT NOT NULL DEFAULT NULL,
+  brand TEXT NOT NULL DEFAULT NULL,
+  model TEXT NOT NULL DEFAULT NULL,
+  itemDescription TEXT NOT NULL DEFAULT NULL,
```
Comment (joeyohie) [inline server/database/setup.sql:20]: I forgot where the team landed on which of parts were required. I thought maybe brand/model was optional. But I guess if you're handling posting of items, can you pass in an empty string if the user doesn't input anything?

File: server/database/setup.sql [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  price NUMERIC(12,2) NOT NULL DEFAULT NULL,
+  nyop BOOLEAN NULL DEFAULT false,
+  min_price NUMERIC(12,2) NULL DEFAULT NULL,
+  availableFrom DATE NOT NULL DEFAULT NULL,
+  availableTo DATE NOT NULL DEFAULT NULL,
+  address1 TEXT NOT NULL DEFAULT NULL,
```
Comment (joeyohie) [inline server/database/setup.sql:26]: I noticed there isn't any addition address fields. Maybe just 'address' instead of 'address1'?

**Review by joeyohie — APPROVED** [review]
Hi Thao! Great work on the schema! I made some comments/observations but you can take them or leave them.

File: server/database/setup.sql (hunk 1 above)
Comment (thaotpnguyen) [reply server/database/setup.sql:34]: Because the index is for item table...column user_id :)

File: server/database/setup.sql (hunk 2 above)
Comment (thaotpnguyen) [reply server/database/setup.sql:20]: I'm thinking to put as N/A as default. I don't want GET to get mix match result for some empty column....

File: server/database/setup.sql (hunk 3 above)
Comment (thaotpnguyen) [reply server/database/setup.sql:26]: address is the reserved word (i think).

**Pull request closed without merging**